docs/01-overview/main-areas/serverless/svls-06-security-considerations.md

```diff
@@ -10,6 +10,10 @@ To eliminate potential security risks when using Functions, bear in mind these f
 
 - Kyma does not define any authorization policies that would restrict Functions' access to other resources within the Namespace. If you deploy a Function in a given Namespace, it can freely access all events and APIs of services within this Namespace.
 
+- Since Kubernetes is [moving from PodSecurityPolicies to PodSecurity Admission Controller](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/), Kyma Functions require running in Namespaces with `baseline` Pod security level. The `restricted` level is not currently supported due to the requirements of the Function building process.
```

Comment:
```suggestion
- Since Kubernetes is [moving from PodSecurityPolicies to PodSecurity Admission Controller](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/), Kyma Functions require running in Namespaces with the `baseline` Pod security level. The `restricted` level is not currently supported due to the requirements of the Function building process.
```